lib/liquid/combinators/tags/capture.ex

```diff
@@ -38,4 +38,22 @@ defmodule Liquid.Combinators.Tags.Capture do
       fn combinator -> optional(combinator, parsec(:__parse__) |> tag(:parts)) end
     )
   end
+
+  def tag2 do
```

Comment:
Why is this function named like this?

Reply:
We want preserve the old tag until we finished doing the improve of performance. Once the proposal is reviewed and accepted, we're going to delete the old tag and rename tag2 -> tag

Reply:
Shouldn't that change be part of accepting this PR?

Reply:
changing it should break some tests which are dependents of reused code